vndr.cfg

```diff
@@ -0,0 +1,11 @@
+github.com/Sirupsen/logrus 26709e2714106fb8ad40b773b711ebce25b78914
+github.com/coreos/go-systemd v4
+github.com/docker/docker 0f5c9d301b9b1cca66b3ea0f9dec3b5317d3686d
```

Comment:
Do you think it'd make sense to move this to a tag too? (the commit referenced here has been in since `v1.8.0-rc1`; https://github.com/docker/docker/commit/0f5c9d301b9b1cca66b3ea0f9dec3b5317d3686d)